components/BlockQuote.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 const BlockQuote = props => (
-    <StyledDiv>
-        <p>
-            {props.children}
-        </p>
-    </StyledDiv>
+  <StyledDiv>
+    <p>{props.children}</p>
+  </StyledDiv>
 );
 
 export default BlockQuote;
 
+BlockQuote.propTypes = {
+  children: PropTypes.object,
```

Comment:
It looks like `children` is a string in the [one place we use it currently](https://github.com/texas-justice-initiative/website-nextjs/blob/68e583730f93aec6bfc5b9ce293796bcb0a15f3b/pages/about.js#L20-L23). What do you think about making that explicit here?

```suggestion
  children: PropTypes.string.isRequired,
```